### ISSUE TITLE
[FEATUREREQUEST] Multi categorys

### ISSUE THREAD

**Issue by apg1980** [Fmstrat/ownnote#10]
the sollution now support only one category.
for example [CATEGORY1] note.htm, [CATEGORY2] note.htm.
it would be awesome to get a more granulary categorism.

suggestion: 
1.) Main category create a subfolder.
2.) Subcategory with the behaviour of [CATEGORY1] note.htm example.

with this it's possible to sort a more fine granulary know how database.

is this possible?

**Comment by Fmstrat** [comment]
I will take a look at this for future versions. I had debating on tagging instead of grouping, but I wanted to make sure this was very simple to use. This could be a good middle ground.

**Comment by mckaygerhard** [comment]
i think its better and more important a category as directory... rather that complicated category system.

a category system will need a database support if u plain supoport and stable api (as i see in u'r code @Fmstrat )

so i think u must think about it , so better make a plain category to directory systeme and let the complicated subcategory system, for very very later using the database for lasted...

**Comment by franciscoluiz** [comment]
i like this feature apg1980... i will use too...

**Comment by ThomasDaheim** [comment]
Not sure if I would want to have a directory tree for categories, sub-categories, ... Especially since you then have two ways of defining a category: by path and by [xx] tag in the file name.
I would prefer to have only one way to do this - either or.
With only directory structure one could e.g. simply move the files to change the category, ... and wouldn't need a UI for all such changes. Maybe that would be a better way than having [cat1][cat2]?

**Comment by mckaygerhard** [comment]
i agree with @ThomasDaheim its more simple to develop and to mantain, also to reflect in owncloud client! that last its more very important
